content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md

```diff
@@ -93,16 +103,40 @@ reproduces the same set of steps that the kubelet performs to calculate
 file-backed memory on the inactive LRU list) from its calculation, as it assumes that
 memory is reclaimable under pressure.
 
-The kubelet recognizes two specific filesystem identifiers:
+The kubelet recognizes three specific filesystem identifiers:
+
+1. `nodefs`: The node's main filesystem, used for local disk volumes,
+    emptyDir volumes not backed by memory, log storage, ephemeral storage,
+    and more. For example, `nodefs` contains `/var/lib/kubelet`.
+
+1. `imagefs`: An optional filesystem that container runtimes can use to store
+   container images (which are the read-only layers) and container writable
+   layers.
+
+1. `containerfs`: An optional filesystem that container runtime can use to
+   store everything else, similarly to what the main filesystem would (see
+   `nodefs`) aside from the container images, which can then be separated (or
+   split from, if you wish) such that the `imagefs` filesystem would only
+   store the images (read-only layers) and nothing else.
+
+As such, kubelet generally allows three options for container filesystems:
+
+- Everything is on the single `nodefs`, also referred to as "rootfs" or
+  simply "root", and there is no dedicated image filesystem.
 
-1. `nodefs`: The node's main filesystem, used for local disk volumes, emptyDir
-   volumes not backed by memory, log storage, and more.
-   For example, `nodefs` contains `/var/lib/kubelet/`.
-1. `imagefs`: An optional filesystem that container runtimes use to store container
-   images and container writable layers.
+- Container storage (see `nodefs`) is on a dedicated disk, and `imagefs`
+  (writable and read-only layers) is separate from the root filesystem.
+  This is often referred to as "Split Disk" (or "Separate Disk") filesystem.
 
-Kubelet auto-discovers these filesystems and ignores other node local filesystems. Kubelet
-does not support other configurations.
+- Container filesystem `containerfs` (same as `nodefs` plus writable
+  layers) is on root and the container images (read-only layers) are
+  stored on separate `imagefs`. This is often referred to as "Split Image"
+  filesystem.
+
+Currently, multiple filesystems for images and containers are not supported.
+
+The kubelet will attempt to auto-discover these filesystems and ignore other
```

Comment:
It would probably be worth mentioning that kubelet auto-discovers this based on the container runtime configuration.

Reply:
Done.